pkg/cmd/cli/cmd/export.go

```diff
@@ -92,9 +94,16 @@ func RunExport(f *clientcmd.Factory, exporter Exporter, in io.Reader, out io.Wri
 	if err != nil {
 		return err
 	}
-	outputVersion, err := kcmdutil.OutputVersion(cmd, clientConfig.GroupVersion)
-	if err != nil {
-		return err
+
+	var outputVersion schema.GroupVersion
+	outputVersionString := kcmdutil.GetFlagString(cmd, "output-version")
+	if len(outputVersionString) == 0 {
+		outputVersion = *clientConfig.GroupVersion
```

Comment:
deprecate output version, select the version you want via your input (`oc export jobs.v1.batch,deployments.v1beta1.apps`, etc)

can we make export output whatever version you requested?